<launch>

  <arg name="server" default="192.168.20.10"/>
  <arg name="frequency" default="10.0"/>

  <node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node" output="screen">
    <rosparam subst_value="true">
      server: $(arg server)
      port: 3883

      update_frequency: 10.0
      frame_id: world

      # Use the VRPN server's time, or the client's ROS time.
      use_server_time: true
      broadcast_tf: false

      # Must either specify refresh frequency > 0.0, or a list of trackers to create
      refresh_tracker_frequency: 1.0
      #trackers:
      #- FirstTracker
      #- SecondTracker
    </rosparam>
      <remap from="/vrpn_client_node/MamboFly/pose" to="/vrpn_client_node/Penne/pose"/>
      <remap from="/vrpn_client_node/MamboFly/twist" to="/vrpn_client_node/Penne/twist"/>
      <remap from="/vrpn_client_node/MamboFly/accel" to="/vrpn_client_node/Penne/accel"/>
  </node>

</launch>